File: backend/codeExecuter/ruby/fizzbuzz/known_correct.rb
def correctUcBes(n)
	results = []
	1.upto n do |i|
		result = ""
		result += "Fizz" if i % 3 == 0
		result += "Buzz" if i % 5 == 0
		result = i.to_s if result == ""
		results[i-1] = result
	end
	return results
end

File: backend/codeExecuter/javascript/ucbes/known_correct.js
let ucBes = (n) => {
  let results = [];
  for (let i = 1; i <= n; i++) {
    let result = "";
    if (i % 3 == 0) {
      result += "Fizz";
    }
    if (i % 5 == 0) {
      result += "Buzz";
    }
    if (result == "") {
      result = i.toString();
    }
    results[i - 1] = result;
  }
  return results;
};

module.exports = {
  ucBes,
};
